src/docfx/data/schemas/learningpath-error-mapping.json

```diff
@@ -0,0 +1,46 @@
+{
+  "properties": {
+    "modules": {
+      "items": {
+        "rules": {
+          "modules": {
+            "xref-not-found": {
+              "severity": "error",
+              "code": "path-child-not-found",
+              "overrideMessage": "Child uid(s): '{0}' can't be found."
```

Comment:
>uid [](start = 40, length = 3)

use capital 'UID(s)' to align to other error message?

Reply:
OK.